docs/incidents/2022/02-03-2022.md

```diff
@@ -0,0 +1,274 @@
+# Incident Report 02-03-2022
```

Comment:
You'll maybe need some front matter in here?

Reply:
Added, thanks.